camect/__init__.py

```diff
@@ -179,6 +180,12 @@ async def _event_handler(self):
                         _LOGGER.debug("Received event: %s", msg)
                         try:
                             evt = json.loads(msg)
+                            if evt['type'] == 'alert':
+                                bout = re.sub(r'^Camera ', '', evt['desc'])
+                                bout = re.sub(r'\.$', '', bout)
+                                bout = bout.split(" just saw a ", 2)
+                                evt['camera'] = bout[0]
+                                evt['object'] = bout[1]
```

Comment:
We have added fields like "cam_name" and "detected_obj" into the API.
Could you take a look and see whether you still need this?